apps/fluent-tester/package.json

```diff
@@ -73,8 +73,8 @@
     "@wdio/local-runner": "^6.5.0",
     "@wdio/spec-reporter": "^6.5.0",
     "@wdio/sync": "^6.5.0",
-    "appium": "1.20.0",
-    "appium-mac2-driver": "0.14.1",
+    "appium": "^2.0.0-beta.27",
```

Comment:
Does "^" work when there's a "-beta" in the name?

Reply:
From https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004:

>^1.2.3-beta.2 := >=1.2.3-beta.2 <2.0.0 Note that prereleases in the 1.2.3 version will be allowed, if they are greater than or equal to beta.2. So, 1.2.3-beta.4 would be allowed, but 1.2.4-beta.2 would not, because it is a prerelease of a different [major, minor, patch] tuple.

So seems like it'll work as long as the tuple doesn't change